packages/common/src/workflow-options.ts

```diff
@@ -253,3 +256,9 @@ export function extractWorkflowType<T extends Workflow>(workflowTypeOrFunc: stri
     `Invalid workflow type: expected either a string or a function, got '${typeof workflowTypeOrFunc}'`
   );
 }
+
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+export function isWorkflowFunctionWithOptions(obj: any): obj is WorkflowFunctionWithOptions<any[], any> {
```

Comment:
Is there some rationale for adding this function in this file/package, even though it is only used from workflow/worker-interface.ts?